app/addons/documents/queryoptions/queryoptions.react.jsx

```diff
@@ -411,7 +405,8 @@ var QueryTray = React.createClass({
             reduce={this.props.reduce}
             toggleReduce={Actions.toggleReduce}
             groupLevel={this.props.groupLevel}
-            updateGroupLevel={Actions.updateGroupLevel} />
+            updateGroupLevel={Actions.updateGroupLevel}
+            docURL={FauxtonAPI.constants.DOC_URLS.GENERAL} />
```

Comment:
The `QueryTray` class doesn't expect docURL as a prop at the moment.  I can update it so that it does, but then it seems as though we're just pushing this same behavior further up the chain of nested React classes.

Cloudant doesn't render `QueryTray` outside of fauxton code, so setting docURL differently (i.e. props versus default value) for each type seems like a lot of effort for minimal return.